vm/adb/adb.go

```diff
@@ -33,8 +33,9 @@ func init() {
 }
 
 type Device struct {
-	Serial  string `json:"serial"`  // device serial to connect
-	Console string `json:"console"` // console device name (e.g. "/dev/pts/0")
+	Serial     string `json:"serial"`      // device serial to connect
+	Console    string `json:"console"`     // console device name (e.g. "/dev/pts/0")
+	ConsoleCmd string `json:"console_cmd"` // command to obtain device console log
```

Comment:
Nit: you could also make it `[]string` to prevent the need to split by the whitespace (which won't work correctly if you need to pass e.g. strings).